### DISCUSSION TITLE
[Help]: Error when dragging far outside the container

### DISCUSSION THREAD

**Discussion by sarussss** [davidjerleke/embla-carousel#777] [Help] [answered]
### Summary

HI @davidjerleke,
When I pull hard on the first slide, it gets stuck and cannot return to the original point, is this a bug or do you want it to work like that?
[Video](https://www.youtube.com/watch?v=02GJzIwZl5w) 

### If applicable, which variants of Embla Carousel are relevant to this question?

- [x] embla-carousel **(Core)**
- [ ] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

### CodeSandbox example

https://codesandbox.io/p/sandbox/embla-carousel-slides-to-scroll-vanilla-9tgj9q?file=%2Findex.html%3A12%2C9

**Comment by davidjerleke** [comment]
@sarussss thanks for your question. I think CodeSandbox is rendering the preview inside an iframe. So when you pull hard, your cursor moves out of the iframe and the iframe never registers a pointer up event. Can you reproduce it here in the [app view](https://9tgj9q.csb.app/)?

**Reply by sarussss** [reply]
Hi @davidjerleke https://youtu.be/xnQiHk722bQ

**Reply by davidjerleke** [reply]
@sarussss I'm not following? My [answer still applies](https://github.com/davidjerleke/embla-carousel/discussions/777#discussioncomment-8629173) because that's the CodeSandbox iframe preview? Additionally, it's related to this:

- https://github.com/davidjerleke/embla-carousel/issues/101#issuecomment-690588094

**Reply by sarussss** [reply]
@davidjerleke Please check again, I put the wrong link  https://youtu.be/xnQiHk722bQ

**Reply by davidjerleke** [reply]
@sarussss thanks 👍! That seems to be a bug yes. Quite hard to reproduce. Do you have a trick to reproduce it easily (it takes like 50 attempts for me to reproduce it)?

**Reply by sarussss** [reply]
Hi @davidjerleke 
Every time I pull it out really far and hold it for a few seconds then let it go, it will happen like that. 
Hope useful for you.

**Reply by sarussss** [reply]
Please drag it into a straight line https://tppr.me/x3NAnG

**Reply by davidjerleke** [reply]
Thanks @sarussss! I will try it soon. Nice catch with the bug 👍🏻.

**Reply by davidjerleke** [reply]
@sarussss it's really weird, I've tried reproducing the bug like you describe but I can only reproduce it like once every 50th try or something. I need to be able to do it more often to be able to find the culprit 😕.

**Reply by sarussss** [reply]
@davidjerleke I tried it and it seems like this error is less common on a large screen, but when I tried it on a 16 inch macbook it was much more likely to encounter this error. You can try to check it on a 16 inch screen or less, hopefully All is well with you.

**Reply by davidjerleke** [reply]
Thanks for the additional tip! I’ll try that.

**Reply by davidjerleke** [reply, marked as answer]
Thanks for all the help @sarussss! This bug is now tracked here:

- #785

Closing this discussion.

Best,
David
